app/controllers/home_controller.rb

```diff
@@ -0,0 +1,6 @@
+class HomeController < ApplicationController
+
+  def dashboard
```

Comment:
it makes sense to me as dashboard as that is what we call it, but for the sake of RESTful-ness